packages/dex/src/out_local.rs

```diff
@@ -257,6 +302,9 @@ mod impl_handler {
                     // TODO do go through RespDelivery
                     Handler::on_open_ica(inner, counterparty_version, deps, env)
                 }
+                State::SwapExactInRecoverIcaRespDelivery(inner) => {
+                    Handler::on_open_ica(inner, counterparty_version, deps, env)
+                }
```

Comment:
I know very well that you are very much against usage of macros, but for such repetitions, I'd strongly suggest it.